[BUG] 1.18 Logblock Error

**Describe the bug**
Whenever someone breaks a tree this spams my console.

**To Reproduce**
Install smoothimber-legacy-1.18.0.jar
Install logblock Build #153 ( https://www.iani.de/jenkins/job/LogBlock/153/ )
Break tree with axe

**Expected behavior**
No errors in log

**Logs**
`java.lang.NoClassDefFoundError: de/diddiz/util/BukkitUtils
        at com.syntaxphoenix.spigot.smoothtimber.compatibility.logblock.LogBlockDatabaseAccessor.isPlayerPlaced(LogBlockDatabaseAccessor.java:48) ~[smoothtimber-legacy-1.18.0.jar:?]
        at com.syntaxphoenix.spigot.smoothtimber.compatibility.logblock.LogBlockResolver.isPlayerPlaced(LogBlockResolver.java:96) ~[smoothtimber-legacy-1.18.0.jar:?]
        at com.syntaxphoenix.spigot.smoothtimber.utilities.locate.Locator.isPlayerPlaced(Locator.java:107) ~[smoothtimber-legacy-1.18.0.jar:?]
        at com.syntaxphoenix.spigot.smoothtimber.listener.BlockBreakListener.onBlockBreak(BlockBreakListener.java:82) ~[smoothtimber-legacy-1.18.0.jar:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor265.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[paper-api-1.18-R0.1-SNAPSHOT.jar:?]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[paper-api-1.18-R0.1-SNAPSHOT.jar:git-Paper-57]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[paper-api-1.18-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:628) ~[paper-api-1.18-R0.1-SNAPSHOT.jar:?]
        at net.minecraft.server.level.ServerPlayerGameMode.destroyBlock(ServerPlayerGameMode.java:400) ~[?:?]
        at net.minecraft.server.level.ServerPlayerGameMode.destroyAndAck(ServerPlayerGameMode.java:354) ~[?:?]
        at net.minecraft.server.level.ServerPlayerGameMode.handleBlockBreakAction(ServerPlayerGameMode.java:314) ~[?:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.handlePlayerAction(ServerGamePacketListenerImpl.java:1744) ~[?:?]
        at net.minecraft.network.protocol.game.ServerboundPlayerActionPacket.handle(ServerboundPlayerActionPacket.java:34) ~[?:?]
        at net.minecraft.network.protocol.game.ServerboundPlayerActionPacket.handle(ServerboundPlayerActionPacket.java:8) ~[?:?]
        at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$1(PacketUtils.java:56) ~[?:?]
        at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper-1.18.jar:git-Paper-57]
        at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:149) ~[?:?]
        at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:23) ~[?:?]
        at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1413) ~[paper-1.18.jar:git-Paper-57]
        at net.minecraft.server.MinecraftServer.c(MinecraftServer.java:189) ~[paper-1.18.jar:git-Paper-57]
        at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:122) ~[?:?]
        at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1391) ~[paper-1.18.jar:git-Paper-57]
        at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1384) ~[paper-1.18.jar:git-Paper-57]
        at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:132) ~[?:?]
        at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1362) ~[paper-1.18.jar:git-Paper-57]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1268) ~[paper-1.18.jar:git-Paper-57]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:317) ~[paper-1.18.jar:git-Paper-57]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.lang.ClassNotFoundException: de.diddiz.util.BukkitUtils
        ... 29 more`

**Server:**
 - Software: Paper
 - Server Version: This server is running Paper version git-Paper-57 (MC: 1.18) (Implementing API version 1.18-R0.1-SNAPSHOT) (Git: 53d026e)
 - Plugin Version: 1.18.0
 - Java Version: 17

--- a/pom.xml
+++ b/pom.xml
@@ -5,7 +5,7 @@
 
 	<groupId>com.syntaxphoenix.spigot</groupId>
 	<artifactId>smoothtimber-legacy</artifactId>
-	<version>1.18.0</version>
+	<version>1.18.1</version>
 	<name>SmoothTimber</name>
 	<packaging>jar</packaging>
 

--- a/src/main/java/com/syntaxphoenix/spigot/smoothtimber/compatibility/logblock/LogBlockDatabaseAccessor.java
+++ b/src/main/java/com/syntaxphoenix/spigot/smoothtimber/compatibility/logblock/LogBlockDatabaseAccessor.java
@@ -13,7 +13,10 @@ import java.sql.Connection;
 import java.sql.PreparedStatement;
 import java.sql.ResultSet;
 import java.sql.SQLException;
+import java.util.Objects;
 import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 /**
  * Used for accessing the logblock database, because the api queries to much not needed information,
@@ -21,9 +24,9 @@ import java.util.Set;
  */
 public class LogBlockDatabaseAccessor {
 
-    private static final Set<Material> EMPTY_MATERIALS = Set.of(
-            Material.AIR, Material.CAVE_AIR, Material.VOID_AIR
-    );
+    private static final Set<Material> EMPTY_MATERIALS = Stream.of(
+            Material.AIR, Material.getMaterial("CAVE_AIR"), Material.getMaterial("VOID_AIR")
+    ).filter(Objects::nonNull).collect(Collectors.toSet());
 
     private final LogBlock logBlock;
 
@@ -44,10 +47,10 @@ public class LogBlockDatabaseAccessor {
             try (ResultSet resultSet = statement.executeQuery()) {
                 if (!resultSet.next())
                     return false;
-                if(resultSet.getInt("playerid") < 0)
+                if (resultSet.getInt("playerid") < 0)
                     return false;
                 Material type = MaterialConverter.getMaterial(resultSet.getInt("type"));
-                if(type.name().endsWith("SAPLING"))
+                if (type.name().endsWith("SAPLING"))
                     return false;
                 return !EMPTY_MATERIALS.contains(type);
             }

--- a/src/main/java/com/syntaxphoenix/spigot/smoothtimber/compatibility/logblock/LogBlockDatabaseAccessor.java
+++ b/src/main/java/com/syntaxphoenix/spigot/smoothtimber/compatibility/logblock/LogBlockDatabaseAccessor.java
@@ -5,7 +5,6 @@ import de.diddiz.LogBlock.LogBlock;
 import de.diddiz.LogBlock.MaterialConverter;
 import de.diddiz.LogBlock.config.Config;
 import de.diddiz.LogBlock.config.WorldConfig;
-import de.diddiz.util.BukkitUtils;
 import org.bukkit.Material;
 import org.bukkit.World;
 import org.bukkit.block.Block;
@@ -14,12 +13,17 @@ import java.sql.Connection;
 import java.sql.PreparedStatement;
 import java.sql.ResultSet;
 import java.sql.SQLException;
+import java.util.Set;
 
 /**
  * Used for accessing the logblock database, because the api queries to much not needed information,
  * which slow things down enormously.
  */
 public class LogBlockDatabaseAccessor {
+
+    private static final Set<Material> EMPTY_MATERIALS = Set.of(
+            Material.AIR, Material.CAVE_AIR, Material.VOID_AIR
+    );
 
     private final LogBlock logBlock;
 
@@ -45,7 +49,7 @@ public class LogBlockDatabaseAccessor {
                 Material type = MaterialConverter.getMaterial(resultSet.getInt("type"));
                 if(type.name().endsWith("SAPLING"))
                     return false;
-                return !BukkitUtils.isEmpty(type);
+                return !EMPTY_MATERIALS.contains(type);
             }
         } catch (SQLException throwables) {
             SmoothTimber.get().getLogger().severe("Failed to connect to LogBlock database");

